static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -257,6 +261,8 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
         return 'mobileMapView'
       case MobileStates.DETAIL_SHOWN:
         return 'mobileDetailShown'
+      default:
```

Comment:
This feels like a default we'd want to surface an error for or check for in the add/remove functions. Otherwise we might end up with `VerticalFullPageMap--` surfacing on the DOM. What do you think?

Reply:
getting to the default here indicates a problem with our code, so I think throwing an error is reasonable